### PR TITLE
Seeding script for fake data

### PR DESCRIPTION
**Pull request by lukehatcher**
Added fake data seeding functionality in database/seed.js and added a "seed" script in my package.json.

### CODE REVIEW

File: database/seed.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    }
+};
+
+const seedingActivities = () => {
+    for (let i = 1; i <= 100; i++) {
+        let data = {id: i, info: []}
+        for (let j = 0; j < 12; j++) {
+            let infoObj = {
+                rating: generateRandFloat(3.5, 5),
+                listing_type: faker.lorem.sentance,
```
Comment (a-heo) [inline database/seed.js]: i think this is supposed to be faker.lorem.sentence

Comment (lukehatcher) [reply database/seed.js]: my second commit "finished seeding script ad391ab" has those changes in them, idk if that's showing?

File: database/seed.js [hunk 2]
```diff
@@ -59,11 +59,11 @@ const seedingActivities = () => {
         for (let j = 0; j < 12; j++) {
             let infoObj = {
                 rating: generateRandFloat(3.5, 5),
-                listing_type: faker.lorem.sentance,
+                activity_name: faker.lorem.sentence(),
```
Comment (a-heo) [inline database/seed.js:62]: do you care about the word range? w/o a range it is going to be between 3-10 letters, something to think about later, but maybe not as important right now.

**Review by a-heo — APPROVED** [review]

File: database/seed.js (hunk 2 above)
Comment (a-heo) [reply database/seed.js:62]: also whoops, i must've looked at your first commit rather than the second one. my bad!

Comment (lukehatcher) [reply database/seed.js:62]: no worries, good idea on the range, i will look into that thx

**Pull request merged**